resources/guides/Navigation Guide/nav-guide-generator.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import JSONData from './sitemap.json'
+
+export const WikiPages = () => {
+  return (
+    <div className='pages-container'>
+      {Object.values(JSONData).map((parent, key) => {
+        return (
+          <div key={key}>
+            <h3>{parent[0].name}</h3>
+            {parent.map((data, key) => {
+              return (
+                <ul key={key}>
+                  <li><a href={data.loc}>{data.name}</a></li>
+                </ul>
+              )
+            })}
+          </div>
+        )
+      })}
+    </div>
+  )
+}
+
+function App() {
```

Comment:
Why not just export the `WikiPages` component? also `App` is too generic of a name.